ppx-lib/gen_js_api_ppx.ml

```diff
@@ -120,7 +126,7 @@ let get_string_attribute_default key default attrs =
   | None -> default
   | Some payload -> payload.attr_loc, id_of_expr (expr_of_payload payload)
 
-let print_error ppf = function
+let _print_error ppf = function
```

Comment:
How come this is not used anymore?

Reply:
Oh I forgot to deal with that :); silly me.